src/redux/session/action/sessionActions.js

```diff
@@ -0,0 +1,53 @@
+export const SIGN_UP_SUCCESS = 'sessions/SIGN_UP_SUCCESS';
+export const SIGN_UP_FAILURE = 'sessions/SIGN_UP_FAILURE';
+export const REFRESH_ACCESS_TOKEN_SUCCESS = 'sessions/REFRESH_ACCESS_TOKEN_SUCCESS';
+export const REFRESH_ACCESS_TOKEN_FAILURE = 'sessions/REFRESH_ACCESS_TOKEN_FAILURE';
+export const LOGIN_SUCCESS = 'sessions/LOGIN_SUCCESS';
+export const LOGIN_FAILURE = 'sessions/LOGIN_FAILURE';
+export const LOGOUT_SUCCESS = 'sessions/LOGOUT_SUCCESS';
+export const LOGOUT_FAILURE = 'sessions/LOGOUT_FAILURE';
```

Comment:
The `session` folder located inside the redux folder should be refactored to have only `sessions.js` file just as with `motorcycles` folder and `reservations` folder in the same redux folder. This would make your redux session have a uniform structure and also make it easily understandable.

So in the end, all actions and reducers for say, user sessions will be found in the `sessions.js` file.